corehq/warehouse/transforms/sql/application_dim.sql

```diff
@@ -0,0 +1,23 @@
+INSERT INTO {{ application_dim }} (
+    application_id,
+    domain,
+    name,
+    application_last_modified,
+    deleted,
+    dim_last_modified,
+    dim_created_on,
+    batch_id
+)
+SELECT
+    application_id,
+    domain,
+    name,
+    application_last_modified,
+    CASE base_doc
+        WHEN 'ApplicationBase' then false
+        WHEN 'ApplicationBase-Deleted' then true
+    END,
+    now(),
+    now(),
```

Comment:
i think Ben intended that to be the timestamp that the dimension record was created, so they're both the same on initial insert